### PR TITLE
Units: Add angle units, Arc Minutes and Seconds

### PR DESCRIPTION
**Pull request by Dripoul**
This PR add 2 angle units (arc-minutes and arc-seconds).

### CODE REVIEW

File: packages/grafana-ui/src/utils/valueFormats/categories.ts [hunk 1]
```diff
@@ -54,6 +54,8 @@ export const getCategories = (): ValueFormatCategory[] => [
       { name: 'Degrees (°)', id: 'degree', fn: toFixedUnit('°') },
       { name: 'Radians', id: 'radian', fn: toFixedUnit('rad') },
       { name: 'Gradian', id: 'grad', fn: toFixedUnit('grad') },
+      { name: 'Arc-minutes', id: 'arcmin', fn: toFixedUnit('arcmin') },
```
Comment (marefr) [inline packages/grafana-ui/src/utils/valueFormats/categories.ts]: Shouldn't units be named `Arc Minutes` and `Arc Seconds`? 

Referring to https://en.wikipedia.org/wiki/Minute_and_second_of_arc

**Review by marefr — COMMENTED** [review]
Looks good. Have a question before we can merge this.

File: packages/grafana-ui/src/utils/valueFormats/categories.ts (hunk 1 above)
Comment (Dripoul) [reply packages/grafana-ui/src/utils/valueFormats/categories.ts]: Yes, should be better.

**Review by marefr — APPROVED** [review]
Nice

**Comment by marefr** [comment]
Thank you for contributing to Grafana!

**Pull request merged**